rust/operator-binary/Cargo.toml

```diff
@@ -1,40 +1,36 @@
 [package]
 authors = ["Lars Francke <lars.francke@stackable.de>"]
 description = "Stackable Operator for druid"
-edition = "2018"
+edition = "2021"
 license = "OSL-3.0"
-name = "stackable-druid-operator-binary"
+name = "stackable-druid-operator"
 repository = "https://github.com/stackabletech/druid-operator"
 version = "0.2.0-nightly"
 build = "build.rs"
```

Comment:
The `build` configuration seems not to be necessary anymore.

Reply:
Oh yes indeed! build.rs is a special name that works without the config line there (https://doc.rust-lang.org/cargo/reference/build-scripts.html)